net-misc/openvswitch/openvswitch-2.17.8.ebuild

```diff
@@ -0,0 +1,152 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{9..12} )
+
+inherit autotools linux-info python-single-r1 systemd tmpfiles
+
+DESCRIPTION="Production quality, multilayer virtual switch"
+HOMEPAGE="https://www.openvswitch.org"
+SRC_URI="https://www.openvswitch.org/releases/${P}.tar.gz"
+
+LICENSE="Apache-2.0 GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~arm64 ~ppc64 ~x86"
```

Comment:
I have no idea how well XDP works on ppc64 (probably OK in skb mode) but until now I really only tested amd64, so the arches should align.

Reply:
I also don't know that,
I have had these improvements active on several AMD64 servers for a long time and without problems
I have left ppc64 because I have not found how to mark XDP blocked for that system